dependencies.json

```diff
@@ -1,124 +0,0 @@
-{
```

Comment:
Pay attention that this file is now deleted on `develop` branch

Reply:
Yes, I deleted on my PR to avoid addition of this file again since it exists no more on master/develop branches